src/bundles/arcade_2d/gameobject.ts

```diff
@@ -1,6 +1,7 @@
 /**
  * This file contains the bundle's representation of GameObjects.
  */
+import Phaser from 'phaser';
```

Comment:
I'm trying to find where this import is used but I can't find it (and yet VSCode shows that it is not an unused import)

Reply:
I am confused by this one to be honest